.github/workflows/deploy-website.yml

```
@@ -0,0 +1,91 @@
+
+env:
+  AZURE_STATIC_WEB_APPS_API_TOKEN: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}
+  AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+  AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+  AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
+  AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+
+jobs:
+  build_and_deploy:
+    name: Build and Deploy Next.js Website
+        runs-on: ubuntu-latest
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)